src/patternfly/components/List/list.scss

```diff
@@ -1,54 +1,41 @@
 @import "../../patternfly-utilities";
 
 .pf-c-list {
-  // Inline modifier variables
-  --pf-c-list--m-inline--MarginRight: var(--pf-global--spacer--md);
-
-  // Grid modifier variables
-  --pf-c-list--m-grid--GridGap: var(--pf-global--spacer--md);
-
-  // Content variables (no modifier)
-  --pf-c-content--li--MarginTop: var(--pf-global--spacer--sm);
-  --pf-c-content--ul--PaddingLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--MarginTop: var(--pf-global--spacer--md);
-  --pf-c-content--MarginBottom: var(--pf-global--spacer--md);
-  --pf-c-content--ul--MarginLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--ListStyle: var(--pf-global--ListStyle);
-  --pf-c-content--ul--nested--MarginTop: var(--pf-global--spacer--sm);
-  --pf-c-content--ul--nested--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-list__item--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-list--m-ul--ListStyle: disc;
+  --pf-c-list--m-ol--ListStyle: decimal;
```

Comment:
Why define this? For the content component's `ol`',s we just let the style come from the browser.

Reply:
I defined this because in a scenario like this: 
`<ul>
	<ol>
		<ul>
		</ul>
	</ol>
</ul> `

The third level of `<ul>` should get a disc as its bullet but it inherits a number for some reason... when I change the list-style to just disc it works. Not sure whats causing this?